api/test/robot_tests/web_on_chain/web_on_chain.robot

```diff
@@ -46,14 +46,33 @@ POST /websiteUploader/upload
 
     Should Be Equal As Strings    ${response.json()['address']}    ${WEBSITE_ADDRESS}
 
-GET /all/domains 
-    ${response}=    GET
-    ...    ${API_URL}/all/domains
-    ...    expected_status=${STATUS_OK}
-    Should Contain    ${response.text}    flappy
-
 GET /my/domains/{nickname}
     ${response}=    GET
     ...    ${API_URL}/my/domains/${WALLET_NICKNAME}
     ...    expected_status=${STATUS_OK}
     Should Contain    ${response.text}    ${website_url}
+
+Fetch Current DNS
+    ${response}=    GET    
+    ...    ${API_URL}/massa/node
+    ...    expected_status=${STATUS_OK}
+    ${current_dns}    Set Variable    ${response.json()['dns']}
+    Set Global Variable    ${DNS}    ${current_dns}
```

Comment:
Should be in the keywords.resource for this test suite, and called in the SetupSuite

Reply:
Or moved into this function : https://github.com/massalabs/station/blob/1034-rf-remove-test-website-at-the-end-of-the-test-suite/api/test/robot_tests/web_on_chain/keywords.resource#L24